web/packages/teleterm/package.json

```diff
@@ -45,9 +45,9 @@
     "@types/whatwg-url": "^13.0.0",
     "@xterm/addon-fit": "^0.10.0",
     "@xterm/xterm": "^5.5.0",
-    "electron": "34.3.0",
-    "electron-builder": "^25.1.8",
-    "electron-vite": "^3.0.0",
+    "electron": "35.1.4",
+    "electron-builder": "^26.0.12",
```

Comment:
The after-install script need to be updated. They have since added my solution for apparmor to it and then improved on it, it'd be better to keep ours in sync with what's in electron-builder.

https://github.com/electron-userland/electron-builder/commits/v26.0.12/packages/app-builder-lib/templates/linux/after-install.tpl